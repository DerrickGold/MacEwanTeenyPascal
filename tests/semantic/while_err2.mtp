const true := 1;
var a : integer;
begin
	while (true) do
	      a := a + 1;

end.