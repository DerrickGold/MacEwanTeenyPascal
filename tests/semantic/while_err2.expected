
Semantic Error [line 4]: value: "true"
	Invalid conditional type: Expected type(s) 'Boolean'; found 'Integer' instead.
